src/content/pages/credits.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Credits
+---
+
+import Panel from 'components/Panel';
+
+<Panel>
+
+
+# Photo credits
```

Comment:
Won't the title be used as the H1 in the template? If not it probably should be! I'd think this should be an H2:
```suggestion
## Photo credits
```

Reply:
Let's do this for now and re-evaluate the title/header setup in a separate PR—we may want some more differentiated styling for page titles too.